include/functions.h

```diff
@@ -2161,7 +2161,7 @@ void func_801159c0(s16 param_1);
 void func_801159EC(s16 arg0);
 void func_80115A14(s32 arg0, s16 arg1);
 // void Parameter_AddMagic(void);
-// void func_80115D5C(void);
+void func_80115D5C(GlobalContext* globalCtx);
```

Comment:
Could this be `GameState*`, I wonder.

Reply:
Throws a warning (because the pointer type is incompatible) but compiles fine and OKs. Should I attempt to cast `globalCtx` like this to make this work?

```
func_80115D5C((GameState*) globalCtx);
```

Not sure if that's better or worse than just keeping `GlobalContext`

Reply:
Use `&globalCtx->state` for now. It'll change later, but that should work.